statics/js/components/rule-detail.js

```diff
@@ -580,78 +581,130 @@ Vue.component('groups-detail', {
+                                    <a data-toggle="tab"\
+                                        role="tab"\
+                                    :href="\'#P\' + tname + \'-\' + port">{{layout.portname(port)}}</a>\
+                                </li>\
+                            </ul>\
+                        </div>\
+                        <div class="tab-content"\
+                            v-if="Object.keys(table.ports).length > 0">\
+                            <div :class="{ active: (pidx==0) }"\
+                                class="tab-pane"\
+                                :id="\'P\' + tname + \'-\' + port"\
+                                role="tabpanel"\
+                                v-for="(rules, port, pidx) in table.ports">\
+                                <rule-table-detail :rules="rules" :layout="layout"/>\
+                            </div>\
+                        </div>\
+                        <rule-table-detail :rules="table.any" :layout="layout"/>\
+                        <div style="background-color: #666; padding: 4px">\
+                            <filter-selector :query="value"\
+                                :filters="filters"\
+                                @add="addFilter"\
+                            @remove="removeFilter"></filter-selector>\
+                        </div>\
+                    </div>\
                 </div>\
-                <ul class="nav nav-pills"\
-                    role="tablist">\
-                    <li :class="{ active: (pidx==0) }"\
-                        v-for="(rules,port,pidx) in table.ports"\
-                        v-on:mouseover="layout.mark(port)"\
-                        v-on:mouseleave="layout.unmark(port)">\
-                        <a data-toggle="tab"\
-                            role="tab"\
-                            :href="\'#P\' + tname + \'-\' + port">{{layout.portname(port)}}</a>\
-                    </li>\
-                </ul>\
-              </div>\
-              <div class="tab-content"\
-                  v-if="Object.keys(table.ports).length > 0">\
-                  <div :class="{ active: (pidx==0) }"\
-                      class="tab-pane"\
-                      :id="\'P\' + tname + \'-\' + port"\
-                      role="tabpanel"\
-                      v-for="(rules, port, pidx) in table.ports">\
-                      <rule-table-detail :rules="rules" :layout="layout"/>\
-                  </div>\
-              </div>\
-              <rule-table-detail :rules="table.any" :layout="layout"/>\
-          </div>\
-      </div>\
-    </div>\
-    <div>\
-      <groups-detail :layout="layout"/>\
+            </div>\
+        <div>\
+            <groups-detail :layout="layout"/>\
+        </div>\
     </div>\
-</div>\
   ',
+
+  components: {
+    'filter-selector': FilterSelector
+  },
+
   props: {
     bridge: {
       type: Object,
       required: true
-    },
-    graph: {
+      },
+    realgraph: {
```

Comment:
I would prefer to use graph here (I know that's me who used real for the draft :)